doc/source/reference/configuration.rst

```diff
@@ -120,30 +120,41 @@ used internally.
 
 .. _plugin-file:
 
-The Plugin File
----------------
+Plugin Files
+------------
 
-The plugin file is a means of creating custom fields, quantities, data
-objects, colormaps, and other code classes and objects to be used in future
+Plugin files are a means of creating custom fields, quantities, data objects,
+colormaps, and other code executable functions or classes to be used in future 
 yt sessions without modifying the source code directly.
 
-To force the plugin file to be parsed, call the function
+To enable a plugin file, call the function
 :func:`~yt.funcs.enable_plugins` at the top of your script.
 
+System wise plugin file
```

Comment:
I think it's worth changing this section header to "global system plugin file", since the next section you have is about local plugin files and there's narrative continuity.

Reply:
done !